docs/release/release_notes/release-0.3.18-(next-release).rst

```diff
@@ -4,3 +4,4 @@ Release 0.3.18 (next release)
 
 Fixes and Features
 ------------------
+* Fix space accounting issue where less cache is taken in consideration when rubix.cache.cleanup.files.during.start was disabled.
```

Comment:
`less cache is taken in consideration` -> `less disk space was considered available and weights of existing files was not picked correctly`